spec/models/info_request_spec.rb

```diff
@@ -2022,6 +2022,32 @@
 
   end
 
+  describe 'updating the URL title' do
+
+    let(:test_requests) do
+      _requests = []
+      3.times do
+        _requests << FactoryBot.create(:info_request, title: 'URL test')
+      end
+      _requests
+    end
+
+    let(:request) do
+      test_requests[1]
+    end
+
+    it 'updates url_title when the title is changed' do
+      request.update_attributes(title: 'Something new')
+      expect(request.url_title).to eq('something_new')
+    end
+
+    it 'does not update url_title when the same title is assigned' do
+      request.update_attributes(title: request.title.dup)
+      expect(request.reload.url_title).to eq('url_test_2')
```

Comment:
Don't need to call `reload` here for the spec to pass, or for the spec to fail without the code change.

Reply:
cheers! left in by mistake while I was trying to pin down what was happening 😞